gulpfile.js

```diff
@@ -1,4 +1,5 @@
 // enable debug for gulp
+require('dotenv').load();
 process.env.DEBUG = process.env.DEBUG || 'fcc:*';
```

Comment:
The require should come after the debug setting. I understand the intent, but the debug should always default to fcc and if the user needs to overwrite it for some reason it should be done on the command line